src/utils/dispatch.js

```diff
@@ -58,19 +74,9 @@ export const createDispatch = (dispatch, state, ...middlewares) => {
  * The function is ispired by thunkMiddelware for redux
  * https://www.npmjs.com/package/redux-thunk
  */
-export const thunkMiddleware = () => {
-	return ({state, dispatch}) => (next) => action => {
+export const thunkMiddleware = ({getState, dispatch}) => (next) => action => {
 		if(typeof action === 'function'){
-			return action({state, dispatch})
+			return action({state: getState(), dispatch})
```

Comment:
state: state vs getState: function(){return knownMiddelwares[identifier].state} zumindest in line 43 dokumentieren?